wrappers/javascript/nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -0,0 +1,674 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
:( temporary?

Reply:
Yeah I believe there were some issues with ref-napi giving some "funky" types. Will do a cleanup PR after this one.